### ISSUE TITLE
HTML/CSS Validator

### ISSUE THREAD

**Issue by osterman** [cloudposse/docs#155]
## what
* Add a HTML/CSS validator to check all docs in `public/` for wellformedness

## why
* We have run into a number of pages built by hugo that are invalid

**Comment by alebabai** [comment]
So, here is my report about this issue.

Regarding **HTML validation**:
In our case we have generated HTML that in most case has very poor formatting.
So many libs recognizes it as non-valid markup (due to bad parser implementation i think).
Another problem is that there are not many npm packages for this purpose. I've tried [htmllint](https://github.com/htmllint), [html-validate](https://html-validate.sidvind.com) and [html-validator](https://github.com/zrrrzzt/html-validator).
First one looks good at all, but it maintained badly - so it has issue and didn't catch many issues in html.
Second one is alse good and fast enough, but had issues with badly-formatted html (recognize it as invalid on `tags-closed` rule - but all tags are closed ).
Last one didn't have option to process directory and validate file be file, and the most reasonable gap is the it uses remote server for validation.
